src/main/scala/com/raphtory/core/components/spout/SpoutExecutor.scala

```diff
@@ -41,6 +43,7 @@ class SpoutExecutor[T](
     executeSpout()
 
   private def executeSpout() = {
+    SpoutTelemetry.totalSpoutReschedules.inc()
```

Comment:
the counter objects cant throw an exception right? i checked the source and i couldn't see anything, if they cant then this is fine.

Reply:
This won't throw an exception since we are only incrementing